addons/point_of_sale/models/account_bank_statement.py

```diff
@@ -10,6 +11,17 @@ class AccountBankStatement(models.Model):
     pos_session_id = fields.Many2one('pos.session', string="Session", copy=False)
     account_id = fields.Many2one('account.account', related='journal_id.default_debit_account_id', readonly=True)
 
+    @api.multi
+    def check_confirm_bank(self):
+        if self.env['pos.session'].statement_ids.search([]).browse(self.id).state  == 'open':
```

Comment:
what is it supposed to do? 'self.env['pos.session'].statement_ids.search([]).browse(self.id)' it really looks like 'self'